File: base/src/main/java/com/pichs/common/base/stack/IStackChild.java
package com.pichs.common.base.stack;

/**
 * 栈的被管理者接口
 */
public interface IStackChild {

    /**
     * 当触发，添加接口
     */
    void onAddActivity();

    /**
     * 当触发，移除接口
     */
    void onRemoveActivity();

}

File: base/src/main/java/com/pichs/base/stack/IStackChild.kt
package com.pichs.base.stack

/**
 * 栈的被管理者接口
 */
interface IStackChild {
    /**
     * 当触发，添加接口
     */
    fun onAddActivity()

    /**
     * 当触发，移除接口
     */
    fun onRemoveActivity()
}